app/assets/javascripts/discourse/tests/unit/models/user-test.js

```diff
@@ -5,13 +5,7 @@ import PreloadStore from "discourse/lib/preload-store";
 import sinon from "sinon";
 import { settled } from "@ember/test-helpers";
 
-module("Unit | Model | user", function (hooks) {
-  hooks.afterEach(function () {
-    if (this.clock) {
-      this.clock.restore();
```

Comment:
Is this a deliberate change? Doesn't seem directly related to this PR? 🤔

Reply:
This is deliberate, yes, this test doesn't contain code that freeze time anymore, so we don't need it.

Reply:
And yes, slightly unrelated to this PR. Tests with frozen time were removed in the main branch.